README.md

```diff
@@ -1,3 +1,4 @@
+[![protected_by hound](https://img.shields.io/badge/protected%20by-Hound-blue.svg)](https://houndci.com/repos)
 Authors Haven - A Social platform for the creative at heart.
```

Comment:
#### Nitpick
Can you please insert a line between the badge and the title. This is still not properly formatted.

<img width="977" alt="screen shot 2018-09-25 at 3 11 03 pm" src="https://user-images.githubusercontent.com/26272628/46020046-574c8600-c0d5-11e8-916f-29941868654a.png">

Reply:
Just added the new line to the readme file. Thanks for the feedback